include/zephyr/drivers/i2c.h

```diff
@@ -599,6 +607,63 @@ static inline int z_impl_i2c_transfer(const struct device *dev,
 	return res;
 }
 
+
+/**
+ * @brief Perform data transfer to another I2C device in master mode.
+ *
+ * This routine provides a generic interface to perform data transfer
+ * to another I2C device asynchronously.
+ *
+ * The array of message @a msgs must not be NULL.  The number of
+ * message @a num_msgs may be zero,in which case no transfer occurs.
```

Comment:
`zero,in` -> `zero, in`

Reply:
removed and replaced with @see to reference the doc string for i2c_transfer